src/utils/sprint-utils.ts

```diff
@@ -0,0 +1,103 @@
+import { Person } from "src/generated/client";
+import { Allocations, Projects, Tasks } from "src/generated/homeLambdasClient";
+
+/**
+ * Retrieve total time entries for an Allocation
+ * 
+ * @param allocation task allocated within a project
+ * @param allocations tasks related to the project
```

Comment:
I'm not sure about this, again allocations are not tasks so not sure what this means. Also can't we just pass in the given allocation (e.g. the first parameter) so why do we also need the list of all allocations?